utils/logging.py

```
@@ -0,0 +1,28 @@
+import os
+import sys
+
+from loguru import logger
+
+
+def setup_loguru(log_dir="logs"):
+    """Configures the Loguru logger for both console and file output.
+
+    This function sets up logging to display messages in the console and to write logs to a file in the specified directory.
```

Pylint (3.3.7)

[convention] Line too long (124/100)

(C0301)